### PR TITLE
Update models.go

### PR DESCRIPTION
**Pull request by pp010**
updated few fields

Thanks for your contribution!
Hi, if there is an issue, that your PR adresses, please link it!

### CODE REVIEW

File: models.go [hunk 1]
```diff
@@ -229,13 +229,14 @@ type ClientScope struct {
 	Description           string                 `json:"description,omitempty"`
 	Protocol              string                 `json:"protocol,omitempty"`
 	ClientScopeAttributes *ClientScopeAttributes `json:"attributes,omitempty"`
-	ProtocolMappers       *ProtocolMappers       `json:"protocolMappers,omitempty"`
+	ProtocolMappers       []ProtocolMappers      `json:"protocolMappers,omitempty"`
```
Comment (pp010) [inline models.go]: Keycloak expects a list

File: models.go [hunk 2]
```diff
@@ -423,8 +425,8 @@ type RealmRepresentation struct {
 	BruteForceProtected                 bool              `json:"bruteForceProtected,omitempty"`
 	ClientAuthenticationFlow            string            `json:"clientAuthenticationFlow,omitempty"`
 	ClientScopeMappings                 map[string]string `json:"clientScopeMappings,omitempty"`
-	ClientScopes                        []interface{}     `json:"clientScopes,omitempty"`
-	Clients                             []interface{}     `json:"clients,omitempty"`
+	ClientScopes                        []ClientScope     `json:"clientScopes,omitempty"`
+	Clients                             []Client          `json:"clients,omitempty"`
```
Comment (pp010) [inline models.go]: Its better to restric the use as required by the apis

File: models.go (hunk 1 above)
Comment (SVilgelm) [reply models.go]: +1

File: models.go (hunk 2 above)
Comment (SVilgelm) [reply models.go]: +1

**Pull request merged**